configs/FLYTEX405V2/config.h

```diff
@@ -106,8 +109,9 @@
 #define DEFAULT_DSHOT_BITBANG DSHOT_BITBANG_OFF
 #define DEFAULT_CURRENT_METER_SOURCE CURRENT_METER_ADC
 #define DEFAULT_VOLTAGE_METER_SOURCE VOLTAGE_METER_ADC
-#define DEFAULT_VOLTAGE_METER_SCALE 110
-#define DEFAULT_CURRENT_METER_SCALE 326
+#define DEFAULT_VOLTAGE_METER_SCALE 105
+#define DEFAULT_VOLTAGE_METER_DIVIDER 5
+#define DEFAULT_CURRENT_METER_SCALE 200
```

Comment:
These values don't make sense. Based on the provided schematic the scale should be 190. Current meter scale should not be set in the config unless the FC is an AIO or sold as part of a stack. 
```suggestion
#define DEFAULT_VOLTAGE_METER_SCALE 190
#define DEFAULT_CURRENT_METER_SCALE 200
```

Reply:
we would like to change divider R3/R6 to bigger value ( 200/10 ) as we are going to use 8S battery. I understand that it is only 11% increase but anyway we have to change initially incorrect value.